launch/pr2_empty_world.launch

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<launch>
+
+  <!-- start up empty world -->
+  <include file="$(find gazebo_ros)/launch/empty_world.launch">
+  </include>
+
+  <!-- Spawn a robot into Gazebo -->
+  <node name="spawn_urdf" pkg="gazebo_ros" type="spawn_model" args="-file $(find access_teleop)/pr2.urdf -urdf -model pr2" />
```

Comment:
I am thinking that we might not want to add this file to master.

Reply:
right